pkg/boilerplate/operator/option.go

```diff
@@ -7,8 +7,7 @@ import (
 )
 
 // key is the singleton key shared by all events
-// the value is irrelevant, but pandas are awesome
-const key = "🐼"
+const key = "service-ca-operator"
```

Comment:
/hold

This is shared code between 3 operators so the value needs to be generic.  I suggest a key value of `熊貓` since the requirement is to remove emoji...

Reply:
Why does it need to be shared across 3 operators? This value shows up in logs, it needs to be informative, not confuse people.

Reply:
It literally is shared - the same exact code is copied between the operators - it is library code.

Reply:
I never got around to trying to move it to lib-go or some other shared spot.

Reply:
Configurable!  I use it for `console-operator`, but every time I see 🐼 I want 🐹 .  Though really it should be 💩 cuz, you know, if this shows up its broken.

Reply:
It'd be great to use a library directly; however no matter where the code lives this key has resulted in confusion and bugs being filed. While the code lives here, we need to update it here to something useful.

Reply:
Here is easy solution: use `o.name`